package.json

```diff
@@ -23,20 +24,24 @@
     "grunt-contrib-uglify": "5.2.2",
     "grunt-contrib-watch": "1.1.0",
     "grunt-sass": "3.1.0",
-    "load-grunt-tasks": "5.1.0"
+    "load-grunt-tasks": "5.1.0",
+    "postcss-cli": "^10.1.0",
+    "tailwindcss": "^3.3.1"
   },
   "scripts": {
     "setup": "npm install --include=dev && composer install && composer dump-autoload -o",
     "postsetup": "grunt",
     "dev": "grunt watch",
     "prebuild": "npm run setup",
-    "build": "grunt build && node .bin/esbuild.mjs"
+    "build": "grunt build && node .bin/esbuild.mjs",
+    "tailwind":"npx tailwindcss -i ./assets/css/tailwind-input-style.css -o ./assets/css/tailwind-output-style.css --watch"
   },
   "engines": {
     "npm": ">=9.5.0",
     "node": ">=18.15.0"
   },
   "dependencies": {
-    "dayjs": "1.11.7"
+    "dayjs": "1.11.7",
+    "flowbite": "^1.6.4"
```

Comment:
What is this for?

Reply:
it is made from while installing tailwind css .

Reply:
Can you send some reference?

Reply:
it is used for date and time , it not installed by tailwind.

Reply:
i had remove this.